app/src/main/java/com/example/ncov19traking/ui/global/GlobalFragment.kt

```diff
@@ -106,13 +118,11 @@ class GlobalFragment : Fragment() {
 
     inner class GlobalFragmentViewHolder {
         private val view = requireView()
-        val totalCases: TextView = view.findViewById(R.id.total_cases)
-        val recovered: TextView = view.findViewById(R.id.recovered_numbers)
-        val deaths: TextView = view.findViewById(R.id.deaths_numbers)
-        val casesPercentage: TextView = view.findViewById(R.id.percentageCasesDifference)
-        val recoveredPercentage: TextView = view.findViewById(R.id.percentageRecoveredDifference)
-        val deathsPercentage: TextView = view.findViewById(R.id.percentageDeathsDifference)
+        val covidTotalCasesCard: CovidCustomCard = view.findViewById(R.id.covid_total_cases_card)
+        val covidRecoveredCard: CovidCustomCard = view.findViewById(R.id.covid_recovered_cases_card)
+        val covidDeathsCard: CovidCustomCard = view.findViewById(R.id.covid_deaths_cases_card)
         val lastUpdateLong: TextView = view.findViewById(R.id.last_update_date)
         val progressBar: ProgressBar = view.findViewById(R.id.progressBar)
+        val pieChart = PieChartBuilder(view.findViewById(R.id.circular_chart))
```

Comment:
mmmm I don't like the naming of this, a `PieChartBuilder` seems like a class who creates piecharts, but this class is not actually building anything is doing configurations in an specific view, you can ignore this method but is something to think about maybe next time we talk I can explain to you what I'm thinking.